adoc/admin-migrate-caasp4.5.adoc

```diff
@@ -0,0 +1,74 @@
+== Migration to {productname} 4.5
+
+1) The OS must be connected to SCC. Otherwise the migration will not work.
```

Comment:
Connecting hosts to the internet is the exception, not the rule in enterprise environments. How would this work with an RMT server ?

Reply:
Well, I suppose in that case there is no automation, and people are setting up the repositories to their server. This is the same in here, so we might need to borrow SLE documentation, and just mention our repositories in addition ?

Reply:
We should at least be explicit about migration with an RMT server.

Reply:
Looking through [zypper-migration-plugin](https://github.com/SUSE/zypper-migration) it seems as if there's no difference between 'connected to SCC' and 'connected to RMT'. Reaching out to the author now for confirmation ...

Reply:
I wasn't the person who tested airgapped clusters. I will have to ask around and/or do it myself again to make sure everything is in line with what is stated here.

Reply:
Just got confirmation from on of the `zypper migration` authors that it works with SCC and RMT. (Event SMT would work but isn't supported with SLE15 afaik).

Bottom line: We can leave the `zypper migration` call and just need to remove the "SCC" parts.
It should be "registration server (like SCC or RMT)". This should cover 'airgap' scenarios well enough.

We should also point to the SLES upgrade guide at https://documentation.suse.com/sles/15-SP2/